### ISSUE TITLE
Error: Several of your accounts are having issues

### ISSUE THREAD

**Issue by hnvn** [nylas/nylas-mail#3589]
OS: MacOS 10.12.5
Nylas version: 1.5.0
Bug: The application shows error: "Several of your accounts are having issues", and keep trying to reconnect again and again. I try to re-authenticate my google account, but nothing changes.

**Comment by hotrush** [comment]
Having this problems with nylas n1 since today's morning for gmail and custom accounts.
Nylas mail works for gmail, but can not sync custom imap accounts.
I think there is a problem with API.

OS Linux mint
Nylas mail 2.0.8

**Comment by austris-argalis** [comment]
Okay, so nylas n1 seems to be discontinued.
You should download nylas mail, which is another client.

https://support.nylas.com/hc/en-us/articles/115005574208-Nylas-Mail-2-0-Release-FAQ

**Comment by hotrush** [comment]
@austris-argalis i have downloaded nylas mail today, it is not syncing my custom imap accounts, they are all empty.

**Comment by jstaf** [comment]
Yep switching to Nylas Mail solved it (was using N1). Thanks for the suggestion @austris-argalis

**Comment by Konzertheld** [comment]
I have the same problem using Nylas Mail, freshly installed. I also get a lot of "error"s "We were unable to apply the changes to your thread, please try again later. Error message: Auth token missing for account".

Also, when I am in the manage accounts window, I unchecked the box for notifying me about news and updates on that account, but the setting is not saved.
